### PR TITLE
feat: Arbitrator 2mb blob test

### PR DESCRIPTION
**Pull request by ethenotethan**
**Description**
- Additional large blob test which asserts that a machine state proof can be generated at an offset > 127kb

Currently 2MB blob pre-images proofs can't be serialized when there's > 1 during execution.

### CODE REVIEW

File: out
Comment (hopeyen) [inline out:1]: for my understanding, what is this file and did you mean to include this?

**Review by hopeyen — APPROVED** [review]
cool:) wish upstream had more comprehensive tests

File: arbitrator/prover/src/main.rs [hunk 1]
```diff
@@ -170,6 +172,13 @@ fn main() -> Result<()> {
             file.read_exact(&mut buf)?;
 
             let hash = hash_preimage(&buf, preimage_ty)?;
+
+            println!(
+                "read preimage of type {:?} with hash 0x{} and size {}",
+                preimage_ty,
+                hex::encode(hash),
+                size
+            );
```
Comment (anupsv) [inline arbitrator/prover/src/main.rs:181]: probably need to remove this.

**Review by bxue-l2 — APPROVED** [review]
lgtm

**Review by anupsv — APPROVED** [review]

**Pull request closed without merging**